concepts/packages/.meta/config.json

```
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
-  "authors": ["ErikSchierboom"],
+  "blurb": "Packages are the unit of code reuse in Go",
```

```suggestion
  "blurb": "Packages are the unit of code reuse in Go.",
```

minor nit: I believe we have been using complete sentences ending in a period.